stable/coredns/templates/serviceaccount.yaml

```diff
@@ -17,4 +17,9 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+{{- if .Values.image.pullSecrets }}
+imagePullSecrets:
+{{- range .Values.image.pullSecrets }}
+  - name: {{ . }}
+{{- end }}
```

Comment:
Same here, missing a second end to close if

Reply:
Fixed and rebased the commits (https://github.com/coredns/helm/pull/36/commits/1df87bc1e980491ac78e6d49a4f287039aa4b333)